timeago-parser/raw/java/CheckAll.java

```diff
@@ -25,13 +25,29 @@ public static void main(String[] args) throws Exception {
 
                 final int size = value.keySet().size();
                 if (size >= 80) {
-                    if (name.equals("seconds")) currentSeconds++;
-                    else if (name.equals("minutes")) currentMinutes++;
-                    else if (name.equals("hours")) currentHours++;
-                    else if (name.equals("days")) currentDays++;
-                    else if (name.equals("weeks")) currentWeeks++;
-                    else if (name.equals("months")) currentMonths++;
-                    else if (name.equals("years")) currentYears++;
+                    switch (name) {
+                        case "seconds":
+                            currentSeconds++
```

Comment:
You should test compile your work. This will not wok because you forgot the semicolon behind your lines.

Reply:
right, thanks

Reply:
Documentation is not yet official tho you can find it here:
https://teamnewpipe.github.io/documentation/00_Prepare_everything/

Reply:
Please always compile and check your code before contributing to a Project. Sending unchecked PR is a no go. :)

Reply:
> And with such an documentation I know why I don't want to.

So you don't like the documentation?
If so I relate, it early as I said, and it's my first try. 
Please be welcome to write suggestions or critic about it [here](https://github.com/teamnewpipe/documentation/issues). It's important that we improve it.